cumulusci/tasks/github_commit.py

```diff
@@ -0,0 +1,188 @@
+import base64
+import hashlib
+import io
+import logging
+import os
+
+from cumulusci.core.exceptions import GithubException
+
+
+class CommitDir(object):
+    """Commit all changes in local_dir to branch/repo_dir"""
+
+    def __init__(self, repo, logger=None, author={}):
```

Comment:
for author, I think you should use a default arg val of None and then check for that in the initializer, since {} is mutable.